### PR TITLE
🎨(up-integration): expanded on academic record entity, added references

### PR DESCRIPTION
**Pull request by Grobbies26**
I added the repository layer in the api to the tsconfig.base.json so that it can be referenced correctly as it did not do so when it was generated, same for the workspace.json

Added the academic record entity, and basic .spec file

Waiting on service and repository layer to finish up their sections to finish API

### CODE REVIEW

File: libs/api/upintegration/api/shared/data-access/src/lib/api-upintegration-academicrecord.spec.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { AcademicRecord } from './api-upintegration-academicrecord.entity';
+
+describe('AvademicRecord', () => {
```
Comment (u20437863-James-Butler) [inline libs/api/upintegration/api/shared/data-access/src/lib/api-upintegration-academicrecord.spec.ts:3]: There is a spelling mistake here

Comment (u20437863-James-Butler) [reply libs/api/upintegration/api/shared/data-access/src/lib/api-upintegration-academicrecord.spec.ts:3]: Just change it to AcademicRecord

Comment (Grobbies26) [reply libs/api/upintegration/api/shared/data-access/src/lib/api-upintegration-academicrecord.spec.ts:3]: I'll have to do that in the next pr, I have loadshedding now or am supposed ti

Comment (u20437863-James-Butler) [reply libs/api/upintegration/api/shared/data-access/src/lib/api-upintegration-academicrecord.spec.ts:3]: Ok

**Review by u20437863-James-Butler — APPROVED** [review]
Just fix the spelling issue in the next PR

**Pull request merged**